### PR TITLE
core: fix dimensions of vecrtor for Array conversion

### PR DESCRIPTION
**Pull request by jcarpent**
Related to #63.
cc @rhaschke seems to solve your issue.

### CODE REVIEW

File: include/eigenpy/details.hpp [hunk 1]
```diff
@@ -117,8 +130,12 @@ namespace eigenpy
     bp::object NumpyMatrixObject; PyTypeObject * NumpyMatrixType;
     //bp::object NumpyAsMatrixObject; PyTypeObject * NumpyAsMatrixType;
     bp::object NumpyArrayObject; PyTypeObject * NumpyArrayType;
+    
+    static NP_TYPE np_type;
```
Comment (rhaschke) [inline include/eigenpy/details.hpp]: I'm not deep into the code. But storing the type (array/matrix) statically doesn't seem to make sense for me on first sight.
Can we still have multiple `ndarray` instances with different type then?

Comment (jcarpent) [reply include/eigenpy/details.hpp]: In fact, with EigenPy, you can return an Eigen::Matrix as either numpy.Matrix or numpy.Array depending on the convention you want to choose. By default, we use the notion of Matrix.

Comment (rhaschke) [reply include/eigenpy/details.hpp]: I'm not sure that this should be a _global_ flag.

Comment (jcarpent) [reply include/eigenpy/details.hpp]: The class is a Singleton, so it can be.

**Pull request merged**